### ISSUE TITLE
Final test target configurations is backwards

### ISSUE THREAD

**Issue by kennebecstars** [kif-framework/KIF#925]
Check the setup of the example red/green app and you will find the documentation is backward.

```
//
// Testable SwiftTests-Shared.xcconfig
//
// Generated by BuildSettingExtractor on 11/11/16
// https://github.com/dempseyatgithub/BuildSettingExtractor
//

**BUNDLE_LOADER = $(TEST_HOST)**
CLANG_ENABLE_MODULES = YES
INFOPLIST_FILE = Testable SwiftTests/Info.plist
LD_RUNPATH_SEARCH_PATHS = $(inherited) @executable_path/Frameworks @loader_path/Frameworks
OTHER_CFLAGS = 
OTHER_LDFLAGS = $(inherited) -framework XCTest -ObjC
PRODUCT_BUNDLE_IDENTIFIER = com.squareup.$(PRODUCT_NAME:rfc1034identifier)
PRODUCT_NAME = $(TARGET_NAME)
SWIFT_OBJC_BRIDGING_HEADER = Testable SwiftTests/Testable SwiftTests-Bridging-Header.h
SWIFT_VERSION = 2.3
**TEST_HOST = $(BUILT_PRODUCTS_DIR)/Testable Swift.app/Testable Swift**
```

**Comment by phatmann** [comment]
Can you point to the errant documentation? And does it matter which variable refers to which?

**Comment by kennebecstars** [comment]
Thanks for your quick reply @phatmann. It is easily fixed. On the readme, section: "Final Test Target Configurations":

Next, configure your bundle loader. In "Build Settings", expand "Linking" and edit "Bundle Loader" to be $(BUILT_PRODUCTS_DIR)/MyApplication.app/MyApplication where MyApplication is the name of your app. Expand the "Testing" section and edit "Test Host" to be $(BUNDLE_LOADER). Also make sure that "Wrapper Extension" is set to "xctest".

Should be:

Next, configure your bundle loader. In "Build Settings", expand "Linking" and edit "Bundle Loader" to be $(TEST_HOST). Expand the "Testing" section and edit "Test Host" to be $(BUILT_PRODUCTS_DIR)/MyApplication.app/MyApplication where MyApplication is the name of your app. Also make sure that "Wrapper Extension" is set to "xctest".

**Comment by phatmann** [comment]
Fixed in 75afbbfd. Thanks.
